sys/net/application_layer/dhcpv6/client.c

```diff
@@ -27,6 +27,7 @@
 #include "xtimer/implementation.h"
 
 #define ENABLE_DEBUG 0
+#define MAX_MUD_URL_LENGTH 0xFF - sizeof(dhcpv6_opt_mud_url_t)
```

Comment:
```suggestion
#define MAX_MUD_URL_LENGTH (0xFF - sizeof(dhcpv6_opt_mud_url_t))
```

Otherwise the result of something like `MAX_MUD_URL_LENGTH * 2` would not be what was commonly expected

Reply:
Thank you, I added the parentheses in 535dd65 .